.github/workflows/build_and_test.yml

```diff
@@ -93,41 +83,59 @@ jobs:
     - name: Check labeler configuration
       run: scripts/check-labeler.pl .github/labeler.yml
 
-    - name: Unit tests
+  schedule:
+    name: Schedule busted tests to run
+    runs-on: ubuntu-22.04
+    needs: build
+
+    env:
+      WORKFLOW_ID: ${{ github.run_id }}
+
+    outputs:
+      runners: ${{ steps.generate-runner-array.outputs.RUNNERS }}
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v4
+
+    - name: Download runtimes file
+      uses: Kong/gh-storage/download@v1
       env:
-        KONG_TEST_PG_DATABASE: kong
-        KONG_TEST_PG_USER: kong
-      run: |
-          source ${{ env.BUILD_ROOT }}/kong-dev-venv.sh
-          TEST_CMD="bin/busted -v -o htest spec/01-unit"
-          if [[ $KONG_TEST_COVERAGE = true ]]; then
-            TEST_CMD="$TEST_CMD --coverage"
-          fi
-          $TEST_CMD
+        GITHUB_TOKEN: ${{ secrets.PAT }}
+      with:
+        repo-path: Kong/gateway-action-storage/main/.ci/runtimes.json
 
-    - name: Archive coverage stats file
+    - name: Schedule tests
+      uses: Kong/gateway-test-scheduler/schedule@5d27a0bf1acc78fab865eafb41d60e0c7625d261 # v2
+      with:
+        test-suites-file: .ci/test_suites.json
+        test-file-runtime-file: .ci/runtimes.json
+        output-prefix: test-chunk.
+        runner-count: ${{ env.RUNNER_COUNT }}
+        static-mode: ${{ github.run_attempt > 1 }}
+
+    - name: Upload schedule files
       uses: actions/upload-artifact@v4
-      if: ${{ always() && (inputs.coverage == true || github.event_name == 'schedule') }}
+      continue-on-error: true
       with:
-        name: luacov-stats-out-${{ github.job }}-${{ github.run_id }}
-        retention-days: 1
-        path: |
-          luacov.stats.out
+        name: schedule-test-files
+        path: test-chunk.*
+        retention-days: 7
 
-    - name: Get kernel message
```

Comment:
Confirmed with @hanshuebner offline and this was removed unintentionally. We should recover the step.

Reply:
this one in particular is ok to be removed, we are only going to keep the one below, because unit and integration tests are executed as part of the same job now (I am going to reintroduce that one).

Reply:
done. https://github.com/Kong/kong/pull/12286/commits/13be66474aa052bb9a9127852e7fe5d163d43262